packages/manager/modules/anthos/package.json

```diff
@@ -12,9 +12,11 @@
   "author": "OVH SAS",
   "main": "./src/index.js",
   "dependencies": {
-    "bootstrap4": "twbs/bootstrap#v4.0.0"
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "lodash": "^4.17.15"
   },
   "peerDependencies": {
+    "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
```

Comment:
Don't forget to update standalone application to add required dependencies

Reply:
Done